### PR TITLE
XFA - Don't fail xfa loading because of a JS subexpression in SOM expressions

### PR DESCRIPTION
**Pull request by calixteman**
- Fix for one pdf in bug 1717668 (PDFIUM-292-0.pdf).

### CODE REVIEW

File: src/core/xfa/bind.js [hunk 1]
```diff
@@ -545,6 +545,12 @@ class Binder {
           // to have something to match with the given expression.
           // See http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.364.2157&rep=rep1&type=pdf#page=199
           match = createDataNode(this.data, dataNode, ref);
+          if (!match) {
+            // For example if the node contains a .(...) then it isn't
+            // findable.
+            // TODO: remove this when .(...) is implemented.
```
Comment (marco-c) [inline src/core/xfa/bind.js:551]: Can you file an issue to track this TODO (and attach a PDF that uses it)?

Comment (calixteman) [reply src/core/xfa/bind.js:551]: https://github.com/mozilla/pdf.js/issues/13688

**Review by marco-c — APPROVED** [review]

**Comment by calixteman** [comment]
@Snuffleupagus, do you know if there is a command to cancel the last running job ?

**Comment by Snuffleupagus** [comment]
> @Snuffleupagus, do you know if there is a command to cancel the last running job ?

No, not as far as I know unfortunately.

You'd have to directly access the bots and *manually* abort the running processes, so it's probably easier to just let them run to completion to avoid the hassle.

**Pull request merged**